packages/rum-angular/package.json

```diff
@@ -27,6 +28,7 @@
     "build": "run-p build:main build:module",
```

Comment:
Why not add `build:es2015` here on `lerna run build`?

Reply:
wow good catch, we used to have `build:*`, totally forgot. thanks for the review @chriscarpenter12  👍

Reply:
added a new PR #736